### PR TITLE
Added missing files to publish process

### PR DESCRIPTION
**Pull request by JarnoVgr**

### CODE REVIEW

**Comment by JarnoVgr** [comment]
For whatever reason Git still doesn't make it easy for me to merge these 3 commits.

**Comment by IntelOrca** [comment]
It's very straight forward. You type git rebase -i then set commits to squash in text file that opens.

**Comment by JarnoVgr** [comment]
Yeah but thanks to stackoverflow I already tried a bunch of methods which screwed it up. git rebase -i gives no 'history' of commits anymore. And it basically says "Rebase 18f04de..18f04de onto 18f04de (1 command(s))" instead of the 3 expected commits.

**Comment by JarnoVgr** [comment]
It's magic. Managed to merge it properly.

File: scripts/ps/publish.ps1 [hunk 1]
```diff
@@ -84,6 +84,8 @@ function Do-Package()
     Copy-Item -Force          "$distDir\changelog.txt"         $tempDir -ErrorAction Stop
     Copy-Item -Force          "$distDir\known_issues.txt"      $tempDir -ErrorAction Stop
     Copy-Item -Force          "$distDir\readme.txt"            $tempDir -ErrorAction Stop
+    Copy-Item -Force          "$rootPath\contributors.md"            $tempDir -ErrorAction Stop
+    Copy-Item -Force          "$rootPath\licence.txt"            $tempDir -ErrorAction Stop
```
Comment (IntelOrca) [inline scripts/ps/publish.ps1]: Please align these correctly, you need to remove a few spaces.

**Comment by JarnoVgr** [comment]
Aligned

**Pull request merged**